File: Python/Manual 01/Question07.py
shape = str(input("Enter C for Circle or S for Square: "))
shape = shape.lower()

if not (shape[0] == "c" or shape[0] == "s"):
    print("Enter a valid Value")
elif shape[0] == "s":
    side = float(input("Enter Length of a Side in cm: "))
    side *= side
    print("Area of your square is: " + str(side) + "cm^2")
else:
    radius = float(input("Enter the radius of your Circle in cm: "))
    radius = 2 * 3.14 * (radius**2)
    print("Area of your Circle is: " + str(radius) + "cm^2")

File: JavaScript/Manual 01/Question07.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter S or s for Square and C or c for Circle: ", function (shape) {
    if (shape == 'S' || shape == 's') {
        rl.question("Enter the side of the Square: ", function (side) {
            console.log("The Area of the Square is: " + (side * side));
            rl.close();
        });
    } else if (shape == 'C' || shape == 'c') {
        rl.question("Enter the radius of the Circle: ", function (radius) {
            console.log("The Area of the Circle is: " + (Math.PI * radius * radius));
            rl.close();
        });
    } else {
        console.log("Invalid Input");
        rl.close();
    }
})